src/main/kotlin/test/ktortemplate/core/exception/ExceptionHandler.kt

```diff
@@ -4,19 +4,21 @@ import io.ktor.application.call
 import io.ktor.features.StatusPages
 import io.ktor.http.HttpStatusCode
 import io.ktor.response.respond
-import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
 fun StatusPages.Configuration.appException() {
+    val log = LoggerFactory.getLogger(this::class.java)
 
     exception<AppException> {
         val errorData = ErrorDTO(it.code.httpStatusCode.value, it.code.messageCode, it.title, it.errors)
-        log.error("Returning AppException data to client:\n$errorData")
+        log.error("Returning AppException data to client:\n$errorData", it)
         call.respond(it.code.httpStatusCode, errorData)
     }
 
     exception<Throwable> {
+        log.error("Returning Throwable", it)
         call.respond(
+            HttpStatusCode.InternalServerError,
```

Comment:
@rmpt I think this was missing, so that exception responses present a 500 status code instead of 200. Also I've added the throwable to the `log.error(message, throwable)` call so we can see the exception in the console.